### ISSUE TITLE
Add example images to `smoothness` field

### ISSUE THREAD

**Issue by tordans** [openstreetmap/iD#8882]
<img width="367" alt="Bildschirmfoto 2022-01-07 um 09 26 59" src="https://user-images.githubusercontent.com/111561/148515493-a26e2437-953f-455b-8b77-e8b2792ed8b6.png">

The `smoothness` key shows a text list of answer right now ([see id-tagging-schema](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/fields/smoothness.json#L6-L17)). 

It would be great to improve the usability and data quality by extending this list. For reference, check out what StreetComplete is working on in https://github.com/streetcomplete/StreetComplete/pull/3617.

Ideally, the UI would…
* show a list of answer options based on the values of the `surface` key
* show the image right in place with the text, similar to what SC does
* show a 1 sentence intro text that scopes the answer, similar to what SC does 

Thinks that we need to think about IMO
* create a new UI
* I think that would be the first field that is dependent on another fields value; also there is the fallback case when `surface` is not present, yet.
* probably extend [the schema-builder](https://github.com/ideditor/schema-builder) so that we can add images and other text to the id-tagging-schema
* maybe use a shared reference source for the list, see idea at https://github.com/streetcomplete/StreetComplete/issues/3592

---

Related to this, https://github.com/openstreetmap/iD/issues/8548 would be great to have as well. Maybe as part of this new UI, we can show a small indicator on the field about the check_date.

**Comment by matkoniecz** [comment]
Not sure whether to move this issue to iD schema builder? Create a new case there linking this one there to enable it?

**Comment by k-yle** [comment]
something like this?
<img width="303" height="191" alt="Image" src="https://github.com/user-attachments/assets/69960a2b-e655-4477-882d-b8ece91f543f" /><sub><em>originaly proposed in https://github.com/openstreetmap/id-tagging-schema/issues/1589#issuecomment-3269609976</em></sub>




maybe we keep this issue for the UI design, and create a new issur for the schema changes? The schema would need to define how we render dropdowns. Either:
1. title only
2. title + description
3. large icon or example image

This schema change would also us to disable #11876 for `access=*`, which a [a bit hard to use now](https://github.com/user-attachments/assets/aad6d35e-f8a1-4648-8f22-28cb144206e1).

**Comment by tordans** [comment]
> something like this?

In general yes; but for this topic `surface` + `smoothness` I think we need the example image that SC uses to make this really nice. This might very well be a case for a separate custom field (that we could even manage in a separate package).

**Comment by maro-21** [comment]
I don't think there is space for images in iD...
There are many images on OSM Wiki...

**Comment by tordans** [comment]
> I don't think there is space for images in iD... 

You men conceptionally or physically? 
- Physically (AKA "is there space", "how does this look") is a UI question which IMO can be solved.
- Conceptionally (AKA "do we want this") is a UX question which IMO is a clear yes, when done right.

SC shows that this can work great when done right in both aspects. And iD can do a lot better to guide users here than it does now.

>There are many images on OSM Wiki...

… which are super hard to find; not well presented and in general not the solution when you want to make quick decisions to review or add data in the editor.

**Comment by maro-21** [comment]
> You men conceptionally or physically?

physically

> which are super hard to find

Nooo...
Images are in the main article: https://wiki.openstreetmap.org/wiki/Key:smoothness
There is also an expanded gallery: https://wiki.openstreetmap.org/wiki/Key:smoothness/Gallery

Personally I find the descriptions of the `smoothness` values in iD - such as which vehicles can pass at a given tag value - more helpful than the photos.

**Comment by tyrasd** [comment]
I would agree that including example photos would be a usability improvement for tags like `surface`, `smoothness`, etc.

The question of how/where to include this concretely in the UI is a different question. There was a little bit of backlash after #11876, when the combobox dropdown did get a bit longer to accommodate tag descriptions. For photos this would be even more relevant, and I do think we need to also consider users with relatively small displays (e.g. notebooks, tablets) where this can be actually a problem. Maybe it would make sense to only expand the options to include the images after a second click by the user…

**Comment by tyrasd** [comment]
PS: @k-yle 

> <img width="303" height="191" alt="Image" src="https://github.com/user-attachments/assets/69960a2b-e655-4477-882d-b8ece91f543f" /><sub><em>originaly proposed in https://github.com/openstreetmap/id-tagging-schema/issues/1589#issuecomment-3269609976</em></sub>

I do like this form of displaying the options even for the icons-only use case… is this a mockup only, or something you have already implemented somewhere?

**Comment by k-yle** [comment]
> is this a mockup only, or something you have already implemented somewhere?

just a mockup :)

**Comment by k-yle** [comment]
> The question of how/where to include this concretely in the UI is a different question. There was a little bit of backlash after https://github.com/openstreetmap/iD/pull/11876, when the combobox dropdown did get a bit longer to accommodate tag descriptions. For photos this would be even more relevant, and I do think we need to also consider users with relatively small displays (e.g. notebooks, tablets) where this can be actually a problem. [...]

A bit off-topic since this issue is about `smoothness`, but another possibility is to break out of the sidebar, and allow the picker to fill the whole screen as a popup. This is definitely overkill for something like `smoothness`, but useful for other image-based fields like traffic signs:

<img width="1128" height="725" alt="Image" src="https://github.com/user-attachments/assets/1175d79b-b8ba-4568-9458-5558dd35a955" />

(screenshot from [the hacky railway signal field](https://kyle.kiwi/iD/#map=20.59/-33.88617/151.20373&id=n1576935726) in a fork of iD)

**Comment by matkoniecz** [comment]
I created https://github.com/openstreetmap/id-tagging-schema/issues/2578 as it requires work in iD tagging schema side

personally I consider it as a good idea, maybe add an option for power users to disable it if interface would reduce work speed for them?

or some smart interface can be found?
